en/cpp/server_plugins.md

```diff
@@ -1,11 +1,26 @@
 # Server Plugins
 
-MAVLink protocols are often designed using the client-server pattern, where vehicle-based components like the autopilot typically act as a server, responding to commands sent by a ground station "client".
+Initially, MAVSDK was developed to act as a ground station (client). MAVSDK would issue ground station commands to an
+autopilot, which would then execute them. This meant MAVSDK always required an external MAVLink autopilot, simulated or real,
+to act as the other-side (server) of the MAVLink service.
 
-Initially MAVSDK was mainly developed with the use case of a ground station in mind, so plugins implemented the client-side protocol behaviour.
+However, MAVSDK can also be used for the server-side of the protocol e.g. running on a companion computer on the vehicle or as an autopilot. This means it is possible to use MAVSDK to act as a vehicle autopilot
```

Comment:
There was something going on with this text. I have rewritten this a bit and generally cleaned up the English. No "factual" changes.

Reply:
Thanks.